src/main/resources/db/migration/V119__batch_court_changes_20.sql

```diff
@@ -0,0 +1,79 @@
+--- FACT-717 ---
+UPDATE public.search_courtaddress
+SET address = E'The Court House\r\nThe Brook',
+    town_name = 'Chatham',
+    postcode = 'ME4 4JZ'
+WHERE id IN (
+    SELECT sca.id
+    FROM public.search_court court
+    JOIN public.search_courtaddress sca
+    ON court.id = sca.court_id
+    JOIN public.search_addresstype sat
+    ON sca.address_type_id = sat.id
+    WHERE slug = 'maidstone-combined-court-centre'
+    AND sat.name = 'Write to us');
+
+--- FACT-721 FACT-722 ---
+UPDATE public.search_court
+SET gbs = 'Y683'
+WHERE slug = 'yeovil-county-family-and-magistrates-court';
+
+UPDATE public.search_court
+SET gbs = 'Y412'
+WHERE slug = 'barnstaple-magistrates-county-and-family-court';
+
+--- FACT-772 ---
+INSERT INTO public.search_courtfacility(id, court_id, facility_id)
+VALUES (DEFAULT,
+        (SELECT id FROM public.search_court WHERE slug = 'nuneaton-county-court'),
+        2379236
+       );
+
+--- FACT-856 ---
+DELETE FROM public.search_courtareaoflaw
+WHERE area_of_law_id = 34249
```

Comment:
@amstevenson in future, can we use get the area_of_law_id field by utilising the area of law name? 
`WHERE area_of_law_id IN (SELECT id FROM public.search_areaoflaw WHERE name = 'Children')`

I always forget to do it that way but it's better for readability in future.